jobs/src/main/scala/dcos/metronome/queue/impl/LaunchQueueServiceImpl.scala

```diff
@@ -0,0 +1,17 @@
+package dcos.metronome.queue.impl
+
+import dcos.metronome.jobrun.impl.QueuedJobRunConverter.QueuedTaskInfoToQueuedJobRunInfo
+import dcos.metronome.model.QueuedJobRunInfo
+import dcos.metronome.queue.LaunchQueueService
+import mesosphere.marathon.core.launchqueue.LaunchQueue
+
+class LaunchQueueServiceImpl(launchQueue: LaunchQueue) extends LaunchQueueService {
```

Comment:
I am not a big fan of "Impl" ... maybe we can go with `MarathonLaunchQueueService` ?

Reply:
I would rather stay consistent with the existing code base for now which already uses "Impl"

Reply:
Agreed with both

my preference is to name the traits "LaunchQueueServiceLike" and them omit from Impl.

But consistency is valuable.